docs/collections/documents/create-document-store.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Create a Document Store
+sidebar_position: 20
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to create a new Document Store collection.
+
+## Create a Document Store Collection with the Console
+
+Follow these instructions to create a new collection using the GDN console web UI.
+
+1. Log in to your Macrometa account.
+1. Click **COLLECTIONS**.
+1. Click **New Collection**.
+1. Enter information about the collection and then click **Create**.
+
+   - **Collection Name -** Required. A unique name to distinguish the collection. Spaces are not allowed.
```

Comment:
Should the hyphen after the item name be bolded?  I changed this earlier as I assumed it was a mistake, but I noticed it's done consistently.

Reply:
Yes, I generally do it that way. We can discuss it if you like, just a preference.

Reply:
No problem, works for me!